webapi_tests/orientation/__init__.py

```diff
@@ -2,5 +2,7 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
+from webapi_tests.orientation.device_motion_test import DeviceMotionTestCommon
+from webapi_tests.orientation.test_device_motion import TestDeviceMotion
```

Comment:
I wonder if it's better to have a new subsuite 'device_motion' instead put in orientation subsuite